doc/nrf/app_dev/board_support/board_names.rst

```diff
@@ -93,6 +93,10 @@ Also see the :ref:`zephyr:boards` section in the Zephyr documentation.
 |                   |            |                                                                   |                                       |
 |                   |            |                                                                   | ``nrf54l15pdk/nrf54l15/cpuapp/ns``    |
 +-------------------+------------+-------------------------------------------------------------------+---------------------------------------+
```

Comment:
Shouldn't PDK be removed at this point?

Reply:
You're right! At least for the NS target of the PDK. (The whole PDK will be removed separately.)

Reply:
Just an FYI - If there still will be some occurrences left in common articles for pdk targets I'll be doing a cleanup closer to the release date